test/stdlib/File_Stat_test.rb

```diff
@@ -0,0 +1,235 @@
+                      File::Stat.new(__FILE__), :inspect
+  end
+
+  def test_mode
+    assert_send_type "() -> Integer",
+                      File::Stat.new(__FILE__), :mode
+  end
+
+  def test_mtime
+    assert_send_type "() -> Time",
+                      File::Stat.new(__FILE__), :mtime
+  end
+
+  def test_nlink
+    assert_send_type "() -> Integer",
+                      File::Stat.new(__FILE__), :nlink
+  end
+
+  def test_owned?
+    assert_send_type "() -> bool",
+                      File::Stat.new(__FILE__), :owned?
+  end
+
+  def test_pipe?
+    assert_send_type "() -> bool",
+                      File::Stat.new(__FILE__), :pipe?
+  end
+
+  def test_rdev
+    assert_send_type "() -> Integer",
+                      File::Stat.new(__FILE__), :rdev
+  end
+
+  def test_rdev_major
+    assert_send_type "() -> Integer",
+                      File::Stat.new(__FILE__), :rdev_major
+  end
+
+  def test_rdev_minor
+    assert_send_type "() -> Integer",
+                      File::Stat.new(__FILE__), :rdev_minor
+  end
+
+  def test_readable?
+    assert_send_type "() -> bool",
+                      File::Stat.new(__FILE__), :readable?
+  end
+
+  def test_readable_real?
+    assert_send_type "() -> bool",
+                      File::Stat.new(__FILE__), :readable_real?
+  end
+
+  def test_setgid?
+    assert_send_type "() -> bool",
+                      File::Stat.new(__FILE__), :setgid?
+  end
+
+  def test_setuid?
+    assert_send_type "() -> bool",
+                      File::Stat.new(__FILE__), :setuid?
+  end
+
+  def test_size
+    assert_send_type "() -> Integer",
+                      File::Stat.new(__FILE__), :size
+  end
+
+  def test_size?
+    assert_send_type "() -> Integer?",
+                      File::Stat.new(__FILE__), :size?
+  end
+
+  def test_socket?
+    assert_send_type "() -> bool",
+                      File::Stat.new(__FILE__), :socket?
+  end
+
+  def test_sticky?
+    assert_send_type "() -> bool",
+                      File::Stat.new(__FILE__), :sticky?
+  end
+
+  def test_symlink?
+    assert_send_type "() -> bool",
+                      File::Stat.new(__FILE__), :symlink?
+  end
+
+  def test_uid
+    assert_send_type "() -> Integer",
+                      File::Stat.new(__FILE__), :uid
+  end
+
+  def test_world_readable?
+    assert_send_type "() -> Integer?",
+                      File::Stat.new(__FILE__), :world_readable?
+  end
+
+  def test_world_writable?
+    assert_send_type "() -> Integer?",
+                      File::Stat.new(__FILE__), :world_writable?
+  end
+
+  def test_writable?
+    assert_send_type "() -> bool",
+                      File::Stat.new(__FILE__), :writable?
+  end
+
+  def test_writable_real?
+    assert_send_type "() -> bool",
+                      File::Stat.new(__FILE__), :writable_real?
+  end
+
+  def test_zero?
+    assert_send_type "() -> bool",
+                      File::Stat.new(__FILE__), :zero?
+  end
+end
```

Comment:
I ran `bin/annotate-with-rdoc core/file.rbs`, but no annotations were added.

I don't know whether this is the correct behavior, or whether there is something wrong with my environment or my operation.